index.d.ts

```diff
@@ -67,6 +67,21 @@ export interface AxiosRequestConfig {
   cancelToken?: CancelToken;
 }
 
+export interface AxiosRequestConfigHeaders {
+  [k: string]: any
```

Comment:
Why not `[k: string]: *string*`

Reply:
for plugin developer can more better `extend interface` in typescript